src/event/handlers/GhostPingUpdateHandler.ts

```diff
@@ -0,0 +1,51 @@
+import {Constants, MessageEmbed, Message, ColorResolvable, MessageActionRow, MessageButton} from "discord.js";
+import EventHandler from "../../abstracts/EventHandler";
+import DateUtils from "../../utils/DateUtils";
+import getConfigValue from "../../utils/getConfigValue";
+import GenericObject from "../../interfaces/GenericObject";
+
+class GhostPingUpdateHandler extends EventHandler {
+	constructor() {
+		super(Constants.Events.MESSAGE_UPDATE);
+	}
+
+	async handle(oldMessage: Message, newMessage: Message): Promise<void> {
+		if (oldMessage.author?.bot) return;
+
+		const oldMentionedUsers = oldMessage.mentions.users;
+
+		if (oldMentionedUsers.size === 0 || oldMentionedUsers.every(user => user.id === oldMessage.author.id || user.bot)) return;
+
+		const removedMentions = oldMentionedUsers.filter(user => {
+			for (const [, newUser] of newMessage.mentions.users) {
+				if (user.id === newUser.id) {
+					return false;
+				}
+			}
+
+			return true;
+		});
+
+		if (removedMentions.size === 0 || removedMentions.every(user => user.id === oldMessage.author.id || user.bot)) return;
+
+		const button = new MessageButton();
+
+		button.setLabel("View Edited Message");
+		button.setStyle("LINK");
```

Comment:
Is there a constant available in Discord.js that we can use here?

Reply:
There is indeed! Did not think to check, will change it to that.